variables.tf

```diff
@@ -183,6 +183,12 @@ variable "pod_mem_limit" {
   description = "The limit of Memory for the Lacework datacollector pod"
 }
 
+variable "lacework_agent_log_stdout" {
+  type        = string
+  default     = "yes"
+  description = "Enable Lacework agent stdout logging"
```

Comment:
Any non-empty value will enable stdout logging. To disable set to empty string ""

Reply:
I would agree, user setting better implemented as a boolean